src/ui-test/utils.ts

```diff
@@ -45,6 +48,24 @@ export async function executeCommand(command: string): Promise<void> {
     throw new Error(`Command '${command}' not found in the command palette`);
 }
 
+/**
+ * Executes a command in the context menu of a specified route.
+ * @param command The command to execute.
+ * @param route The route where the context menu should be opened.
+ * @returns A Promise that resolves when the command is executed.
+ * @throws An error if the specified command is not found in the context menu of the route.
+ */
+export async function executeCommandInContextMenu(command: string, route: string): Promise<void> {
+    const item = await (await new SideBarView().getContent().getSection('resources')).findItem(route) as ViewItem;
+    const menu = await item.openContextMenu();
```

Comment:
if you will divide this method into eg `openContextMenu(route)` and selectContextMenuItem(command) 

you can reuse openContextMenu method in previous two tests verifying menu item is available